src/test/steps/common.ts

```diff
@@ -221,18 +222,24 @@ When('I enter my valid case reference and valid access code', async () => {
 });
 
 When('a case worker issues the application', async () => {
-  await triggerCaseWorkerEvent('caseworker-issue-application', { ceremonyPlace: 'Somewhere' });
+  await triggerAnEvent('caseworker-issue-application', { ceremonyPlace: 'Somewhere' });
 });
 
 When('a case worker updates court case hearing', async () => {
-  await triggerCaseWorkerEvent('system-update-case-court-hearing', {
+  await triggerAnEvent(SYSTEM_UPDATE_CASE_COURT_HEARING, {
```

Comment:
You've imported this event url but not the one above, I don't have a preference either way but better to be consistent